x-pack/test/api_integration/apis/monitoring/elasticsearch/fixtures/index_detail.json

```diff
@@ -1104,146 +1089,143 @@
+      "primary": false,
       "relocating_node": null,
       "shard": 4,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": true,
+      "index": "avocado-tweets-2017.10.02",
       "node": "jUT5KdxfRbORSCWkb5zjmA",
+      "primary": true,
       "relocating_node": null,
       "shard": 4,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": true,
+      "index": "avocado-tweets-2017.10.02",
       "node": "xcP6ue7eRCieNNitFTT0EA",
+      "primary": true,
       "relocating_node": null,
       "shard": 1,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": false,
+      "index": "avocado-tweets-2017.10.02",
       "node": "jUT5KdxfRbORSCWkb5zjmA",
+      "primary": false,
       "relocating_node": null,
       "shard": 1,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": true,
+      "index": "avocado-tweets-2017.10.02",
       "node": "xcP6ue7eRCieNNitFTT0EA",
+      "primary": true,
       "relocating_node": null,
       "shard": 2,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": false,
+      "index": "avocado-tweets-2017.10.02",
       "node": "jUT5KdxfRbORSCWkb5zjmA",
+      "primary": false,
       "relocating_node": null,
       "shard": 2,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": false,
+      "index": "avocado-tweets-2017.10.02",
       "node": "xcP6ue7eRCieNNitFTT0EA",
+      "primary": false,
       "relocating_node": null,
       "shard": 3,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": true,
+      "index": "avocado-tweets-2017.10.02",
       "node": "jUT5KdxfRbORSCWkb5zjmA",
+      "primary": true,
       "relocating_node": null,
       "shard": 3,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": false,
+      "index": "avocado-tweets-2017.10.02",
       "node": "xcP6ue7eRCieNNitFTT0EA",
+      "primary": false,
       "relocating_node": null,
       "shard": 0,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     },
     {
-      "state": "STARTED",
-      "primary": true,
+      "index": "avocado-tweets-2017.10.02",
       "node": "jUT5KdxfRbORSCWkb5zjmA",
+      "primary": true,
       "relocating_node": null,
       "shard": 0,
-      "index": "avocado-tweets-2017.10.02"
+      "state": "STARTED"
     }
   ],
   "shardStats": {
     "nodes": {
       "jUT5KdxfRbORSCWkb5zjmA": {
-        "shardCount": 38,
-        "indexCount": 20,
+        "shardCount": 5,
```

Comment:
I'm pretty sure these are different because the new query now limits the shard data to the specific index instead of across all indices